app/views/bulkrax/importers/show.html.erb

```diff
@@ -0,0 +1,117 @@
+<%# OVERRIDE Bulkrax 5.0 to show errored entries link %>
```

Comment:
Is this something that we should just add to Bulkrax?

I suspect yes.  But that doesn't block this PR.

(Just thinking how much I love not copying views into downstream applications)

Reply:
There was a Bulkrax [PR](https://github.com/samvera-labs/bulkrax/pull/698), however we have some additional work to do before it should be included in Bulkrax. So until then, since this was previously pointing to that branch (errored-entries-tab), we decided to make an override for now.